.dockerignore

```diff
@@ -0,0 +1 @@
+.tmp/
```

Comment:
Great idea! I always had to move .tmp away and back before/after build manually. Thanks!

Reply:
No problem, also we should probably look for other files which shouldn't be added to docker image.